doc/command_ref.rst

```diff
@@ -770,6 +770,16 @@ Remove Command
     There are also a few specific remove commands ``remove-n``, ``remove-na`` and ``remove-nevra``
     that allow to specify exact argument NEVRA format.
 
+Remove Examples
+---------------
+
+``dnf remove acpi tito``
+    Remove packages ``acpi`` and ``tito``
+
+``dnf remove $(dnf repoquery --extras --exclude=tito,acpi)``
+    Remove packages not present in any repository but it does't mark for removal packages ``tito``
```

Comment:
* `Remove packages ... but it doesn't mark` is wrong sentence.
* `if they requires on of removing packages` is wrong sentence as well...